jlm/rvsdg/traverser.hpp

```diff
@@ -7,11 +7,18 @@
 #ifndef JLM_RVSDG_TRAVERSER_HPP
 #define JLM_RVSDG_TRAVERSER_HPP
 
+#include <jlm/rvsdg/tracker.hpp>
```

Comment:
this is strange -- the file now includes itself?

Reply:
The file is called `jlm/rvsdg/traverser.hpp` and the file I include is called `jlm/rvsdg/tracker.hpp`. It is not the same file.

Reply:
sorry yes you are right I misread!